src/components/DataTable/Table.js

```diff
@@ -82,7 +82,7 @@ const TableCell = ({
 		<DateTime
 			className="data-table-datetime"
 			value={obj[header.key]}
-			format={header.datetime.format}
+			dateTimeFormat={header?.datetime?.dateTimeFormat ? header?.datetime?.dateTimeFormat : "medium"}
```

Comment:
can the attribute be just `format` ?

Reply:
@ateska Unfortunately, no. Because that library "date-fns" has a method with the same name `format` and this causes conflicts and the code does not work

Reply:
hmms, that's weird. OK.